components/ai_chat/resources/page/components/main/index.tsx

```diff
@@ -185,16 +188,25 @@ function Main() {
     triggerCharacter: '/',
   })
 
+
+
   return (
     <main
       data-testid='main'
       className={classnames({
         [styles.main]: true,
         [styles.mainPanel]: !aiChatContext.isStandalone,
-        [styles.mainMobile]: aiChatContext.isMobile
+        [styles.mainMobile]: aiChatContext.isMobile,
+        [styles.dragOver]: isDragOver
       })}
       ref={setMainElement}
     >
+      <DragOverlay
+        isDragActive={isDragActive}
+        isDragOver={isDragOver}
+        clearDragState={clearDragState}
+        processDroppedImages={conversationContext.processDroppedImages}
```

Comment:
not critical, but maybe the `DragOverlay` should just read these directly from `useConversationContext` so we don't need to drill the props here?

Reply:
Fixed in 429230ea6bdfe1d310aab08741c35edb74adabda